src/containers/shared/components/Transaction/AMMCreate/test/mock_data/amm_create.json

```diff
@@ -0,0 +1,199 @@
+      },
+      {
+        "ModifiedNode": {
+          "FinalFields": {
+            "Account": "rwRGF9pmfEGT4GcZ379cYC9p3wpJDozy8w",
+            "Balance": "989999999970",
+            "Flags": 8388608,
+            "OwnerCount": 2,
+            "Sequence": 4
+          },
+          "LedgerEntryType": "AccountRoot",
+          "LedgerIndex": "83A4BB5EA0DB3CFEE89804BC08E1CBD222E869046EE01A7ECCAB57F8433F9655",
+          "PreviousFields": {
+            "Balance": "999999999980",
+            "OwnerCount": 1,
+            "Sequence": 3
+          },
+          "PreviousTxnID": "F7998834E30DEB92E62754744328C882CD1C50C59A7568884B14F498B2B91198",
+          "PreviousTxnLgrSeq": 114136
+        }
+      },
+      {
+        "ModifiedNode": {
+          "FinalFields": {
+            "Flags": 0,
+            "Owner": "rhpHaFggC92ELty3n3yDEtuFgWxXWkUFET",
+            "RootIndex": "A49DF2BCB7DE1E35F4F0662624A45DF35D56BC3833008E0BE8721424BBD80E33"
+          },
+          "LedgerEntryType": "DirectoryNode",
+          "LedgerIndex": "A49DF2BCB7DE1E35F4F0662624A45DF35D56BC3833008E0BE8721424BBD80E33"
+        }
+      },
+      {
+        "CreatedNode": {
+          "LedgerEntryType": "DirectoryNode",
+          "LedgerIndex": "A55924837E8BDDBAA2A59B4E1490E81CA63EC5266C679FF5418F1975AA793ADF",
+          "NewFields": {
+            "Owner": "rMEdVzU8mtEArzjrN9avm3kA675GX7ez8W",
+            "RootIndex": "A55924837E8BDDBAA2A59B4E1490E81CA63EC5266C679FF5418F1975AA793ADF"
+          }
+        }
+      },
+      {
+        "CreatedNode": {
+          "LedgerEntryType": "AMM",
+          "LedgerIndex": "C7FD06649235AF4CABD8FA6D8BB0CAF6C6EA5038A74D0DDD5025290683636D02",
+          "NewFields": {
+            "AMMAccount": "rMEdVzU8mtEArzjrN9avm3kA675GX7ez8W",
+            "Asset2": {
+              "currency": "USD",
+              "issuer": "rhpHaFggC92ELty3n3yDEtuFgWxXWkUFET"
+            },
+            "LPTokenBalance": {
+              "currency": "03930D02208264E2E40EC1B0C09E4DB96EE197B1",
+              "issuer": "rMEdVzU8mtEArzjrN9avm3kA675GX7ez8W",
+              "value": "10000000"
+            }
+          }
+        }
+      },
+      {
+        "ModifiedNode": {
+          "FinalFields": {
+            "Flags": 0,
+            "Owner": "rwRGF9pmfEGT4GcZ379cYC9p3wpJDozy8w",
+            "RootIndex": "D69B2631EAE1115AFA8A12284B3780D743CB6CBA74A2F5D878CD68C0352C4CFA"
+          },
+          "LedgerEntryType": "DirectoryNode",
+          "LedgerIndex": "D69B2631EAE1115AFA8A12284B3780D743CB6CBA74A2F5D878CD68C0352C4CFA"
+        }
+      },
+      {
+        "ModifiedNode": {
+          "LedgerEntryType": "AccountRoot",
+          "LedgerIndex": "F2C157AFC0B58AE61CF96D0A671A35B0E240E3BFBCE71F18604307627AB1EDF5",
+          "PreviousTxnID": "B05CAE1B66F6A64ADB5B7753395E49FC8E5376D3E95ED50BE5F7395509D6334C",
+          "PreviousTxnLgrSeq": 114138
+        }
+      }
+    ],
+    "TransactionIndex": 0,
+    "TransactionResult": "tesSUCCESS"
+  }
+}
```

Comment:
super nit: there should be a newline at the end of all the JSONs

Reply:
Agreed

Reply:
why? what difference does it make? it's a test file that will never be read by human eyes and the parser has no issues with no new lines at the end of the file.

can you explain why this is important in any capacity? @ckniffen @mvadari

Reply:
Because every file we have it.  Helps with git compares.

Reply:
if we're Git comparing this file to a previous version of itself both wont have the new line so it it'll look the same. Is there another reason? 

I'm really trying to understand why we even have this practice in the first place. It seems unnecessary.

Reply:
https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e

I guess our linter isnt catching it

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

ill make this change